pypeit/display/display.py

```diff
@@ -666,3 +666,24 @@ def show_scattered_light(image_list, slits=None, wcs_match=True):
         if clear:
             clear = False
 
+
+def show_1dspec(filename, ext=0, masked=True, fluxed=False, extraction='OPT'):
```

Comment:
It doesn't look like any of the keyword arguments are used.  Is this a mistake or something left to a future PR?

Reply:
Good catch--overlooked TBD, I will push an update to handle these.

Reply:
It now handles the command line parameters.